yktrace/src/swt.rs

```diff
@@ -16,8 +16,7 @@ impl ThreadTracerImpl for SWTThreadTracer {
         if buf as usize == 0 {
             Err(InvalidTraceError::InternalError)
         } else {
-            // When we make a SWTSirTrace, we convert all of the locations from core::SirLoc up to
-            // crate::SirLoc and store them in self so that we can hand out references via raw_loc().
+            // When we make a SirTrace, we convert all of the locations from SwtLoc to SirLoc.
```

Comment:
I hadn't heard of `SwtLoc` (I thought it was called `core::SirLoc`). I went searching for both of these names in the compiler and found nothing. My guess is that it's been removed and that this code probably doesn't compile.

I guess that's no surprise. We are not testing the (vastly unimplemented) software tracing backend.

My recommendation is to keep the comment as it was, just remove `via raw_loc()`. Then an interested party can at least grep for `core::SirLoc` in the compiler git history.

Reply:
`core::SirLoc` hasn't existed anymore for a long time. `SwtLoc` is defined lower in this file. I introduced it when I added software tracing support on the Yorick side again for use with my experiments with cg_clif as compiler a while ago: https://github.com/softdevteam/yk/pull/108

Reply:
Gotcha